addons/im_livechat/static/src/legacy/public_livechat_chatbot.js

```diff
@@ -441,8 +443,8 @@ const QWeb = core.qweb;
                 return false;
             } else if ((this._chatbotIsExpectingUserInput() ||
                         this._chatbotCurrentStep.chatbot_step_type === 'question_selection') &&
-                       this._messages.length !== 0) {
-                const lastMessage = this._messages[this._messages.length - 1];
+                        this.messaging.livechatButtonView.messages.length !== 0) {
+                const lastMessage = this.messaging.livechatButtonView.messages[this.messaging.livechatButtonView.messages.length - 1];
```

Comment:
that indentation is hellish lol
how about putting `) {` to the next line, so that it looks a bit less weird?

Reply:
Ok, will do.
TBH I don't really care of the shape of the code inside legacy code.
As long as things are moved to models, sooner or later widgets will cease to exist.

Reply:
Changes done